blas/unit_test/Test_Blas2_syr2.hpp

```diff
@@ -1685,7 +1685,7 @@ void Syr2Tester<ScalarX, tLayoutX, ScalarY, tLayoutY, ScalarA, tLayoutA,
   // ********************************************************************
   // Compare
   // ********************************************************************
-  this->compareKkSyr2AgainstReference(alpha, h_A_syr2, h_ger_reference.h_view);
+  this->compareKkSyr2AgainstReference(alpha, h_A_syr2, h_ger_reference.d_view);
```

Comment:
I wouldn't expect the d_view needed for comparison with other apparent host h_ views

Reply:
Yep, my bad this will probably crap out at runtime... let's fix that quickly!